src/fast_array_utils/stats/_sum.py

```diff
@@ -24,7 +29,9 @@ def sum_(
     *,
     axis: Literal[0, 1, None] = None,
     dtype: DTypeLike | None = None,
+    keep_cupy_as_array: bool = False,
 ) -> NDArray[Any] | np.number[Any] | types.CupyArray | types.DaskArray:
+    del keep_cupy_as_array
```

Comment:
why do you `del` this? so it explicitly can't be used below?

Reply:
Yeah, it’s an explicit “this parameter isn’t used” marker.